containers/cobol-gnucobol-2.2.0/test-project/test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Could there be guidance in the readme to explain how to use the test script?

Reply:
I've removed those script tests.